newIDE/app/src/VariablesList/index.js

```diff
@@ -334,9 +348,12 @@ export default class VariablesList extends React.Component<Props, State> {
           useDragHandle
           lockToContainerEdges
         >
+          {!objVariablesTree.length && this._renderEmpty()}
```

Comment:
This will render an empty message if there are no object variables, which is something that we don't want as far as I see.

Reply:
I scrapped that approach altogether